### PR TITLE
Instance segmentation project

### PR DESCRIPTION
**Pull request by mecevit**

### CODE REVIEW

File: instance-segmentation/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+```python
+import layer
+import torch
+
+
+my_model = layer.get_model("layer/instance-segmentation/models/object_detector:4.2").get_train()
+my_model.eval()
+
+img = ... # Your tensor image
```
Comment (mwitiderrick) [inline instance-segmentation/README.md:31]: I think you can update this to include the image/tensor here https://github.com/layerai/examples/blob/mecevit/instance-segmentation/instance-segmentation/notebooks/demo.ipynb

**Review by mwitiderrick — APPROVED** [review]

**Comment by mwitiderrick** [comment]
Looks amazing, apart from the partnership article, can you write a tutorial about this?

File: instance-segmentation/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+cython==0.29.30
+pycocotools==2.0.4
+torchvision
+segments-ai==0.70
```
Comment (mwitiderrick) [inline instance-segmentation/requirements.txt:4]: Do we need Layer here?

File: instance-segmentation/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
+# COCO Instance Segmentation
+
+[![Open in Layer](https://development.layer.co/assets/badge.svg)](https://app.layer.ai/layer/instance-segmentation/) [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/layerai/examples/blob/main/instance-segmentation/notebooks/demo.ipynb) [![Layer Examples Github](https://badgen.net/badge/icon/github?icon=github&label)](https://github.com/layerai/examples/tree/main/instance-segmentation)
```
Comment (volkangurel) [inline instance-segmentation/README.md:3]:
```suggestion
[![Open in Layer](https://app.layer.ai/assets/badge.svg)](https://app.layer.ai/layer/instance-segmentation/) [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/layerai/examples/blob/main/instance-segmentation/notebooks/demo.ipynb) [![Layer Examples Github](https://badgen.net/badge/icon/github?icon=github&label)](https://github.com/layerai/examples/tree/main/instance-segmentation)
```

**Review by volkangurel — CHANGES_REQUESTED** [review]

File: instance-segmentation/requirements.txt (hunk 2 above)
Comment (mecevit) [reply instance-segmentation/requirements.txt:4]: It breaks the remote build because of Layer
